dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -314,6 +317,34 @@ public void testComponent_linkageCheckResult() throws IOException, ParsingExcept
                 + " referenced from 21 classes ▶"); // '▶' is the toggle button
   }
 
+  @Test
+  public void testComponent_linkageCheckResult_java11() throws IOException, ParsingException {
+    String javaVersion = System.getProperty("java.version");
+    // javaMajorVersion is 1 when we use Java 8. Still good indicator to ensure Java 11 or higher.
+    int javaMajorVersion = Integer.parseInt(javaVersion.split("\\.")[0]);
+    Assume.assumeTrue(javaMajorVersion >= 11);
+
+    // version used in libraries-bom 1.0.0
+    Document document = parseOutputFile(
+        "com.google.http-client_google-http-client-appengine_1.29.1.html");
+    Nodes reports = document.query("//p[@class='jar-linkage-report']");
+    Assert.assertEquals(2, reports.size());
+    // This number differs between Java 8 and Java 11
```

Comment:
This should be explained here, not just in the PR description.

Reply:
Yes, updated.